client/src/app/queries/sboms.ts

```diff
@@ -113,13 +116,16 @@ export const useFetchSbomsByPackageId = (
 ) => {
   const { data, isLoading, error, refetch } = useQuery({
     queryKey: [SBOMsQueryKey, "by-package", packageId, params],
-    queryFn: () => getSBOMsByPackageId(packageId, params),
+    queryFn: () =>
+      dataOf(
+        listRelatedSboms({ client, path: { id: packageId }, body: params })
```

Comment:
This change is breaking the current UI.
To reproduce it:
- Go to the packages list page
- Click on any package and you will be redirected to the package details page
- Click on the `SBOMS` tab
- Open your Browser dev tools and you will see a request like:
```
GET http://localhost:3000/api/v1/sbom/by-package
```

That is wrong as the expected request should be something like:
```
GET http://localhost:3000/api/v1/sbom/by-package?limit=10&offset=0&id=85901454-4c44-5572-b3b6-7dba8aefb3e3
```

Reply:
Thx for the heads up.  Will look into this more.

Reply:
If it helps, I think we can reuse this block of code https://github.com/trustification/trustify-ui/blob/main/client/src/app/api/rest.ts#L39-L42 

That creates all the QueryParameters that axios needs/receives and appends to a request